typings/react-datetime-tests.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+import { Moment } from 'moment';
```

Comment:
I prefer using ES6 syntax, the code is cleaner and allows for better static analysis. It is also the preferred style for Typescript.